doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -193,7 +194,15 @@ See :ref:`cmake_options` for more information about defining CMake options.
 Performing prerequisite operations
 **********************************
 
-You must enable the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option to support the Google Fast Pair standard in your project.
+To add the Google Fast Pair subsystem to your project, you must enable the ``SB_CONFIG_BT_FAST_PAIR`` Kconfig option if you use the default |NCS| build system configuration with sysbuild.
```

Comment:
Could you create a dedicated section: `Kconfig` with a dedicated link to this section? _ug_bt_fast_pair_prerequisite_ops still seems broad